tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino

```diff
@@ -1,7 +1,7 @@
 ARG UBUNTU_VERSION=20.04
 FROM ubuntu:${UBUNTU_VERSION}
 
-ARG OPENVINO_VERSION=2023.0.0
+ARG OPENVINO_VERSION=2023.3.0
 ARG PYTHON_VERSION=3.8
```

Comment:
Can we change the Python Version to 3.9 here. Will CI support it

Reply:
it's driven by the dockerfile. if it's ubuntu 20.04 the default python3 version is 3.8 , but you can install other versions.